build failure on node 4: invalid peer dependency chai

```
npm WARN peerDependencies The peer dependency chai@>= 2.0.0 < 4 included from chai-immutable will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN optional dep failed, continuing fsevents@1.1.3
npm ERR! Linux 4.9.6-040906-generic
npm ERR! argv "/home/travis/.nvm/versions/node/v4.8.6/bin/node" "/home/travis/.nvm/versions/node/v4.8.6/bin/npm" "install"
npm ERR! node v4.8.6
npm ERR! npm  v2.15.11
npm ERR! code EPEERINVALID
npm ERR! peerinvalid The package chai@4.1.2 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer chai-immutable@1.6.0 wants chai@>= 2.0.0 < 4
npm ERR! peerinvalid Peer sinon-chai@2.14.0 wants chai@>=1.9.2 <5
```

For example see https://travis-ci.org/politie/sherlock/jobs/301581188

--- a/package.json
+++ b/package.json
@@ -56,7 +56,7 @@
         "@types/sinon": "^2.3.7",
         "@types/sinon-chai": "^2.7.29",
         "chai": "^4.1.2",
-        "chai-immutable": "^1.6.0",
+        "chai-immutable": "^2.0.0-alpha.1",
         "coveralls": "^3.0.0",
         "immutable": "^3.8.2",
         "mocha": "^4.0.1",

